File: grpc-server/src/main/java/com/unla/grupoF/entities/Evento.java
package com.unla.grupoF.entities;

import javax.persistence.*;
import lombok.*;
import java.time.LocalDateTime;
import java.util.HashSet;
import java.util.Set;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Evento {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(nullable = false)
    private String nombreEvento;

    private String descripcion;

    @Column(nullable = false)
    private LocalDateTime fechaHora;

    @ManyToMany
    @JoinTable(
        name = "evento_usuarios",
        joinColumns = @JoinColumn(name = "evento_id"),
        inverseJoinColumns = @JoinColumn(name = "usuario_id")
    )
    private Set<Usuario> miembros = new HashSet<>();
}

File: graphql-server/models/Evento.py
from sqlalchemy import Column, Integer, String, DateTime
from db import Base

class Evento(Base):
    __tablename__ = "evento"
    id = Column(Integer, primary_key=True, index=True)
    descripcion = Column(String)
    fecha_hora = Column(DateTime)
    nombre_evento = Column(String)